### PR TITLE
choice: update page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** latest

## Changes

- Simplify examples and command description.
- Command examples reordered: from simple to complex. Complexity is measured in arguments count.

### CODE REVIEW

File: pages/windows/choice.md [hunk 1]
```diff
@@ -1,29 +1,28 @@
...
+`choice /cs /c {{Ab}}`
 
-- Specify a message to display before the list of choices. If `/m` is not specified, only the choice prompt is displayed:
+- Prompt user to select a choise and prefer the [d]efault choise in a specific [t]ime:
 
-`choice /m {{message}} /C {{ABC}}`
+`choice /t {{5}} /d {{N}}`
 
-- Display help message:
+- Display the help:
```
Comment (CleanMachine1) [inline pages/windows/choice.md:26]:
```suggestion
- Display help:
```

File: pages/windows/choice.md [hunk 2]
```diff
@@ -1,29 +1,28 @@
...
+`choice /m "{{message}}"`
 
-- Specify the number of seconds to pause before using the default choice specified by `/d`:
+- Prompt user to select a [c]ase-[s]ensitive [c]hoice from a specific set:
 
-`choice /C {{AaBb}} /t {{3}} /d {{b}}`
+`choice /cs /c {{Ab}}`
 
-- Specify a message to display before the list of choices. If `/m` is not specified, only the choice prompt is displayed:
+- Prompt user to select a choise and prefer the [d]efault choise in a specific [t]ime:
```
Comment (CleanMachine1) [inline pages/windows/choice.md:22]:
```suggestion
- Prompt user to select a choise and prefer the [d]efault choice in a specific [t]ime:
```

File: pages/windows/choice.md [hunk 3]
```diff
@@ -1,29 +1,28 @@
...
+`choice`
 
-- Use the default [Y,N] list of choices:
+- Prompt user to select a [c]hoice from a specific set:
 
-`choice`
+`choice /c {{AB}}`
 
-- Specify that the choices are case-sensitive:
+- Prompt user to select a choise with a specific [m]essage:
```
Comment (CleanMachine1) [inline pages/windows/choice.md:14]:
```suggestion
- Prompt user to select a choice with a specific [m]essage:
```

File: pages/windows/choice.md [hunk 4]
```diff
@@ -1,29 +1,28 @@
 # choice
 
-> Prompts the user to select one item from a list of single-character choices in a batch program, and then returns the index of the selected choice.
-> If used without parameters, choice displays the default choices Y and N.
+> Prompt user to select a choise and return the selected choice index.
 > More information: <https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/choice>.
 
-- A,B and C as list of choices to be used:
+- Prompt user to select `Y` or `N` choice:
```
Comment (mfrw) [inline pages/windows/choice.md:6]: Not sure if I am correct or not, but i feel like we should have a `the` before the `user`:
```suggestion
- Prompt the user to select `Y` or `N` choice:
```

I think this comment, if correct should be applied to all IMO

Comment (EmilySeville7cfg) [reply pages/windows/choice.md]: Done. I've used `the current user` instead of just `the user`.

File: pages/windows/choice.md (hunk 1 above)
Comment (EmilySeville7cfg) [reply pages/windows/choice.md]: Done.

File: pages/windows/choice.md [hunk 5]
```diff
@@ -1,29 +1,28 @@
 # choice
 
-> Prompts the user to select one item from a list of single-character choices in a batch program, and then returns the index of the selected choice.
-> If used without parameters, choice displays the default choices Y and N.
+> Prompt user to select a choise and return the selected choice index.
```
Comment (blueskyson) [inline pages/windows/choice.md:3]:
```suggestion
> Prompt user to select a choice and return the selected choice index.
```

File: pages/windows/choice.md [hunk 6]
```diff
@@ -1,29 +1,28 @@
...
+`choice /m "{{message}}"`
 
-- Specify the number of seconds to pause before using the default choice specified by `/d`:
+- Prompt the current user to select a [c]ase-[s]ensitive [c]hoice from a specific set:
 
-`choice /C {{AaBb}} /t {{3}} /d {{b}}`
+`choice /cs /c {{Ab}}`
 
-- Specify a message to display before the list of choices. If `/m` is not specified, only the choice prompt is displayed:
+- Prompt the current user to select a choise and prefer the [d]efault choice in a specific [t]ime:
```
Comment (CleanMachine1) [inline pages/windows/choice.md:22]:
```suggestion
- Prompt the current user to select a choice and prefer the [d]efault choice in a specific [t]ime:
```

File: pages/windows/choice.md [hunk 7]
```diff
@@ -1,29 +1,28 @@
 # choice
 
-> Prompts the user to select one item from a list of single-character choices in a batch program, and then returns the index of the selected choice.
-> If used without parameters, choice displays the default choices Y and N.
+> Prompt user to select a choice and return the selected choice index.
 > More information: <https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/choice>.
 
-- A,B and C as list of choices to be used:
+- Prompt the current user to select `Y` or `N` choice:
```
Comment (CleanMachine1) [inline pages/windows/choice.md:6]:
```suggestion
- Prompt the current user to select a `Y` or `N` choice:
```

**Review by CleanMachine1 — APPROVED** [review]

**Review by Nico385412 — APPROVED** [review]
LGTM

**Review by kbdharun — APPROVED** [review]
LGTM

**Pull request merged**